testpilot-metrics.js

```diff
@@ -95,18 +96,25 @@ Metrics.prototype = {
    * should return an object whose keys are GA Measurement Protocol parameters.
    * The returned object will be form encoded and sent to GA.
    */
-  sendEvent: function({method, object=null, category='interactions', variant=null} = {}, transform) {
+  sendEvent: function({method, object, category, variant} = {}, transform) {
```

Comment:
I set a default empty parameter object, to avoid throwing if sendEvent was accidentally called with no arguments (one of the basic goals here is for sendEvent never to emit an Error, only to log them).

Unfortunately, seems like setting a default parameter object _and_ setting default parameter values does not actually work, because the category was getting defaulted to 'undefined' in GA pings.